crates/bevy_ecs/src/query/filter.rs

```diff
@@ -366,7 +366,7 @@ macro_rules! impl_query_filter_tuple {
                     if _not_first {
```

Comment:
Same here.

Reply:
Removed the comment and renamed variables a bit to avoid confusion with the "intersect" word